### PR TITLE
Create chroot with desktop

### PR DESCRIPTION
**Pull request by ghost**
Cusdeb-single-node will create chroot with xfce4 for speed build image.

### CODE REVIEW

**Comment by denismosolov** [comment]
@leocoan could you please think how do we can pass the config file?

**Comment by ghost** [comment]
@denismosolov i think that unnecessary when we create only chroot-env

File: functions.sh [hunk 1]
```diff
@@ -351,6 +351,26 @@ build_env() {
...
+
+            pushd "${TARGET}"/pieman
+                for chroot in "${chroot_desktop[@]}"; do
+                    IFS=',' read -r -a pieces <<< "${chroot}"
+                    env CREATE_ONLY_CHROOT=true \
+                        OS="${pieces[0]}" \
+                        PROJECT_NAME="${pieces[0]}-${pieces[1]}" \
+                        DEVICE="${pieces[2]}" \
+                        PYTHON="${TARGET}"/pieman-env/bin/python \
+                        XFCE4="true" \
```
Comment (denismosolov) [inline functions.sh]: Let's use `XFCE4=true` instead of `XFCE4="true"` for consistency

**Comment by denismosolov** [comment]
@leocoan 
It interrupts the installation
Selection_670

**Comment by ghost** [comment]
@denismosolov are you install debconf-util package on your system?

**Comment by denismosolov** [comment]
@leocoan yes, I've installed debconf-util.
I can confirm the issues I mentioned is not related to this PR. But we will need to fix Pieman

File: functions.sh [hunk 2]
```diff
@@ -351,6 +351,26 @@ build_env() {
...
+
+            pushd "${TARGET}"/pieman
+                for chroot in "${chroot_desktop[@]}"; do
+                    IFS=',' read -r -a pieces <<< "${chroot}"
+                    env CREATE_ONLY_CHROOT=true \
+                        OS="${pieces[0]}" \
+                        PROJECT_NAME="${pieces[0]}-${pieces[1]}" \
+                        DEVICE="${pieces[2]}" \
+                        PYTHON="${TARGET}"/pieman-env/bin/python \
+                        XFCE4=true \
```
Comment (denismosolov) [inline functions.sh:368]: @leocoan I guess where would be a problem when we add more desktop environments with `XFCE4=true`. Would be great if you can suggest how we can update this code if we add `GNOME=true` for example.

Comment (ghost) [reply functions.sh:368]: It's unnecessary. We will copy CREATE_ONLY_CHROOT block and set XFCE4=false and CNOME(and other)=true

**Review by denismosolov — APPROVED** [review]

**Pull request merged**